### PR TITLE
Address more review feedback

### PR DESCRIPTION
**Pull request by wolfs**

### CODE REVIEW

File: contents/introduction.adoc [hunk 1]
```diff
@@ -11,45 +11,48 @@ Gradle’s build cache is a versatile feature that can be used in different ways
...
 
 === Share results between CI builds
 
 The build cache can do more than going back-and-forth in time: it can also bridge physical distance between computers, allowing results generated on one machine to be re-used by another.
 A usual first step when introducing the build cache within a team is to enable it for builds running as part of _continuous integration_ only. Using a shared HTTP build cache backend (such as https://gradle.com/build-cache/[the one provided by Gradle Enterprise]) can significantly reduce the work CI agents need to do.
-This translates into faster feedback to developers, and less money spent on the CI infrastructure.
+This translates into much faster feedback to developers, and less money spent on the CI infrastructure.
+Faster builds also mean less commits being part of each build which makes debugging issues much more efficient.
```
Comment (lptr) [inline contents/introduction.adoc]: less -> fewer

Comment (lptr) [inline contents/introduction.adoc]: Comman before "which"

File: contents/introduction.adoc [hunk 2]
```diff
@@ -11,45 +11,48 @@ Gradle’s build cache is a versatile feature that can be used in different ways
...
+It allows you to easily find out which build produced an artifact coming from the build cache by clicking on the corresponding icon in the timeline:
 
 [.screenshot]
 image::from-cache-origin.png[]
 
 === Accelerate developer builds by reusing CI results
 
 When multiple developers work on the same project, they don't just need to build their own changes: whenever they pull from version control, they end up having to build each others' changes as well.
-If the pulled changes are sufficiently independent, the developer can safely re-use outputs already generated on CI.
+If the current developer is working on something independent of the pulled changes, he can safely re-use outputs already generated on CI.
```
Comment (lptr) [inline contents/introduction.adoc]: I'm not sure "current" means anything in this context. Why not just say "Whenever a developer is working on something independent of the pulled changes, they can safely..."

Also: don't assume developers are male.

File: contents/performance.adoc [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
-The sole reason to use any build cache is to make builds faster. But how much faster can you go when using the cache? Measuring the impact is both important and complicated, as cache performance is determined by many factors. Initial measurements of the cache's impact can validate the extra effort (work, infrastructure) that was required to start using the cache. These measurements can later serve as baselines for future improvements, and to watch for signs of regressions.
+The sole reason to use any build cache is to make builds faster.
+But how much faster can you go when using the cache?
+Measuring the impact is both important and complicated, as cache performance is determined by many factors.
+Initial measurements of the cache's impact can validate the extra effort (work, infrastructure) that was required to start using the cache.
+These measurements can later serve as baselines for future improvements, and to watch for signs of regressions.
+
+[NOTE]
+====
+Performance gains may vary wildly between a properly configured and maintained build cache and build and one that is not.
```
Comment (lptr) [inline contents/performance.adoc]: Missing comma before "and".

Comment (lptr) [inline contents/performance.adoc]: I would rephrase this to: "Proper configuration and maintenance of a build can improve caching performance in a big way."

File: contents/performance.adoc [hunk 4]
```diff
@@ -41,6 +53,8 @@ It's important to distinguish those parts of the pipeline that caching has no ef
 You can use https://confluence.jetbrains.com/display/TCD10/Statistic+Charts[statistic charts] if you use Teamcity for your CI builds.
 Most of time you will end up extracting data from your CI server via the corresponding REST API (see https://wiki.jenkins-ci.org/display/JENKINS/Remote+access+API[Jenkins remote access API] and https://confluence.jetbrains.com/display/TCD10/REST+API[Teamcity REST API]).
 When using Gradle Enterprise, you can use the https://docs.gradle.com/enterprise/export-api/[export API] to access the necessary data and run your analytics on them.
+The data which can be obtained from Gradle Enterprise is be much richer compared to what can be provided by CI servers.
```
Comment (lptr) [inline contents/performance.adoc]: which -> that

But in general active over passive is better, something like:

> Gradle Enterprise provides richer data than can be obtained from the CI server.

File: contents/performance.adoc [hunk 5]
```diff
@@ -41,6 +53,8 @@ It's important to distinguish those parts of the pipeline that caching has no ef
...
+For example, you can even get insights into the execution of single tasks.
```
Comment (lptr) [inline contents/performance.adoc]: I think adding "even" here weakens the message.

File: contents/introduction.adoc (hunk 1 above)
Comment (wolfs) [inline contents/introduction.adoc]: fixed

File: contents/performance.adoc (hunk 3 above)
Comment (wolfs) [inline contents/performance.adoc]: Rephrased.

File: contents/performance.adoc (hunk 4 above)
Comment (wolfs) [inline contents/performance.adoc]: Rephrased.

File: contents/performance.adoc (hunk 5 above)
Comment (wolfs) [inline contents/performance.adoc]: Removed "even".

**Comment by wolfs** [comment]
@lptr Could you have another look?
